src/qt/forms/optionsdialog.ui

```diff
@@ -206,6 +206,16 @@
           <string>Expert</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
+          <item>
+           <widget class="QCheckBox" name="subFeeFromAmount">
+            <property name="toolTip">
+             <string extracomment="This is a toolTip, Located at Options dialog under the Wallet tab. Select default setting for subtract the fee from the amount checkbox while creating new recipients.">Default setting to whether Subtact Fee From Amount or not.</string>
+            </property>
+            <property name="text">
+             <string extracomment="Located at Options dialog under the Wallet tab. Select default setting for subtract the fee from the amount checkbox while creating new recipients.">Subtract &amp;Fee From Amount</string>
```

Comment:
```suggestion
             <string extracomment="An options menu setting to always subtract the fee from the amount a user is sending.">Subtract &amp;Fee From Amount</string>
```

Reply:
@psancheti110 updated strings to remove the word always as it is not appropriate